docs/source/conf.py

```diff
@@ -161,6 +163,47 @@
 html_js_files = ["js/custom.js"]
 
 
+# Configure viewcode extension.
+code_url = "https://github.com/kornia/kornia/blob/main"
```

Comment:
this way the docs will always redirect to the main, should we someway as scikit inject the commit/revision here? https://github.com/scikit-learn/scikit-learn/blob/main/doc/sphinxext/github_link.py

Reply:
really love this feature. Add the reference link to the file in case we want to add / remove stuff